packages/client/src/components/Markdown/renderers/LinkRenderer.tsx

```diff
@@ -23,7 +23,12 @@ const LinkRenderer: React.FC<LinkRendererProps> = ({
   const isPhoneLink = url.protocol === "tel:";
 
   // Pass the text in link (eg: <a>text in link</a>) or `TEXT_LINK` as fallback
-  const { value = eventNames.TEXT_LINK } = children[0]?.props;
+  const { value } = children[0]?.props || "";
```

Comment:
Maybe change `""` with `{}`?
This seems very weird to me `const {value} = ""`.
`const {value} = {}` makes more sense.
Or `const value = children[0]?.props?.value` even more readable.

Reply:
Fixed. Also the file is even more improved